examples/example.py

```diff
@@ -0,0 +1,62 @@
+import streamlit as st
```

Comment:
Is this file needed?

(We're trying to get rid of all examples that already have an equivalent e2e test)